src/mqt/predictor/rl/helper.py

```diff
@@ -421,3 +435,84 @@ def get_path_trained_model():
 
 def get_path_training_circuits():
     return get_path_training_data() / "training_circuits"
+
+
+def load_model(model_name: str):
+    path = get_path_trained_model()
+
+    if Path(path / (model_name + ".zip")).exists():
+        return MaskablePPO.load(path / (model_name + ".zip"))
+
+    logger.info("Model does not exist. Try to retrieve suitable Model from GitHub...")
+    try:
+        mqtpredictor_module_version = metadata.version("mqt.predictor")
+    except ModuleNotFoundError:
+        raise RuntimeError(
+            "Could not retrieve version of mqt.predictor. Please run 'pip install . or pip install mqt.predictor'."
+        ) from ModuleNotFoundError
```

Comment:
```suggestion
    except ModuleNotFoundError:
        raise RuntimeError(
            "Could not retrieve version of mqt.predictor. Please run 'pip install . or pip install mqt.predictor'."
        ) from None
```
is the pythonic way to re-raise exceptions. A good article on that: https://stefan.sofa-rockers.org/2020/10/28/raise-from/

Reply:
Done.